packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -218,6 +220,27 @@ class TextSpan extends InlineSpan implements HitTestTarget, MouseTrackerAnnotati
   /// ```
   final String? semanticsLabel;
 
+  /// The language of the text in this span and its span children.
+  ///
+  /// Setting the locale of this text span affects the way that assistive
+  /// technologies, such as VoiceOver or TalkBack, pronounce the text.
+  ///
+  /// If this span contains other text span children, they are also inherited
+  /// the locale from this span unless explicitly set to different locales.
+  final ui.Locale? locale;
+
+  /// Whether the assistive technologies should spell out this text character
+  /// by character
+  ///
+  /// If the text is 'hello world', setting this to true causes the assistive
+  /// technologies, such as VoiceOver or TalkBack, to pronounce
+  /// 'h-e-l-l-o-space-w-o-r-l-d' instead of complete words. This is useful for
+  /// texts, such as passwords or verification codes.
+  ///
+  /// If this span contains other text span children, they are also inherited
+  /// the property from this span unless explicitly set.
```

Comment:
Nit: Up to you, but maybe consider adding a sentence about the default behavior if this isn't specified.